packages/frontend-2/components/settings/workspaces/members/actions/UpdateSeatTypeDialog.vue

```diff
@@ -32,20 +34,29 @@ import {
 import { useWorkspaceUpdateSeatType } from '~/lib/workspaces/composables/management'
 import { useWorkspacePlan } from '~/lib/workspaces/composables/plan'
 import SeatTransitionCards from './SeatTransitionCards.vue'
-import type {
-  SettingsWorkspacesMembersActionsMenu_UserFragment,
-  SettingsWorkspacesMembersTableHeader_WorkspaceFragment
-} from '~/lib/common/generated/gql/graphql'
+import type { SettingsWorkspacesMembersTableHeader_WorkspaceFragment } from '~/lib/common/generated/gql/graphql'
 import { Roles } from '@speckle/shared'
 import { useMixpanel } from '~~/lib/core/composables/mp'
 
+type UpgradeSeatTypeDialogUser = {
```

Comment:
This is a quick fix to allow for this modal to be used on the collaborators page, as there is no exact fragment match. We should look into a way to decouple these upgrade modals from the settings, as they will be used in other places as well